### DISCUSSION TITLE
Longruner WS2812B led matrix

### DISCUSSION THREAD

**Discussion by abl73** [pixelit-project/PixelIt#67] [Q&A] [answered]
First of all, thanks for this amazing project 👍.
Got a question when assembling.
Bought the following items:
- AZDelivery D1 Mini NodeMcu mit ESP8266-12F WLAN Module
- Longruner WS2812B led matrix
- Elko Kondensator Radial 1000µF 35V 105°C

Got the D1 up and running and webpage can be reached.
Connected the display to the same power adapter and can see the two upper left leds burning. 

The problem is that no time is displayed (if I read correctly it should display that when restarted) nor can I send any test texts.

Anyone could give me a hint what I might be doing wrong (read somewhere that the display needs its own power supply, does that mean I need to have two power supplies connected)?

**Comment by o0shojo0o** [comment, marked as answer]
Hi, which binary did you flash?

**Reply by abl73** [reply]
I flashed the ESP8266.bin because the others didn't work.
I thought the problem was caused by my power-adapter (tried it on a DC5V 450mA and didn't know the matrix needed far more A), so ordered a new one today...

**Reply by o0shojo0o** [reply]
Yes the power supply is too weak, but I would still try the **firmware_d1_mini.bin**.

**Reply by abl73** [reply]
I will try the new power supply tomorrow and will also try to flash the d1_mini again. Furthermore, I will start from a clean sheet as I started with some trouble related to not using the proper USB cable which was only a power cable and no data one. Thereafter I tried to install several alternative drivers which might have messed it all up...

**Comment by abl73** [comment]
Power supply arrived (5V10A) and I connected it to the led strips.
Also tried to flash the d1 mini firmware as suggested. What I discovered is that I downloaded the firmware twice yesterday that added (1) to the name of the firmware. Apparently that is not accepted, when I deleted the (1) it flashed correctly (had to first use the clean flash).

All connected now, webserver is running, but the led strip shows some distortions. With the settings 'Type 2' I can recognize some characters PixelIT and 03 09 and 19 50 but part of the screen is not correct. Any suggestions I could follow?

**Comment by abl73** [comment]
Reconnected it all (had the ESP powered by USB and the LED matrix directly at the power supply). Now all is working and I can start experimenting :-)...

**Comment by GreeTzZ** [comment]
Hi,
I got the same issue but wasn't able to solve it.
Would you please let me know which power supply you bought?
Furthermore, I would be happy if you share as much information about your setting at all, e.g. circuit diagram. 

Many Thanks in advance & BR
GreeTzZ

**Reply by abl73** [reply]
Hi,
I bought a DC 5V 10A power supply (if you search for "5v 10a power supply" on Amazon you'll find multiple of them).
As a non-technical person I tried to the best I could to draw a picture with the component and wiring.
Hope this helps,
Arno
![PixelIT](https://user-images.githubusercontent.com/68024401/134785850-89b3f168-decc-4175-ad83-81e0ae1ab79a.PNG)

**Comment by abl73** [comment]
One last thing where I would seek some help is the fact that every so often, the brightness of the display gets really high (and leds turn green) for a second or two and then it normalizes again. Any suggestion what could be the cause?

**Reply by PricelessToolkit** [reply]
Hi, you soldered resistor between D1 and Matrix?
